### PR TITLE
Add new theme Eyed

### PR DESCRIPTION
**Pull request by febryars33**
Hello,
This is the first theme I made for phpMyAdmin

Originally from https://github.com/phpmyadmin/themes/tree/master/bootstrap and https://github.com/phpmyadmin/themes/tree/master/blueberry for some icons

Can you review this ? 😄

### CODE REVIEW

**Comment by febryars33** [comment]
https://raw.githubusercontent.com/febryars33/eyed-phpmyadmin-theme/main/screen.png

File: eyed/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Security Policy
```
Comment (williamdes) [inline eyed/SECURITY.md:1]: can you remove the root files that are not related to the theme ?
SECURITY, CODE_OF_CONDUCT, .gitignore

File: eyed/theme.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Eyed",
+    "version": "5.1",
```
Comment (williamdes) [inline eyed/theme.json:3]:
```suggestion
    "version": "1.0.0",
```
Would be best ?

File: eyed/theme.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    "description": "Eyed theme for phpMyAdmin",
+    "author": "Febriansyah Riki Setiadi",
+    "url": "https://github.com/febryars33/eyed-phpmyadmin-theme/",
+    "supports": [
+        "5.1",
```
Comment (williamdes) [inline eyed/theme.json:8]: 5.1 works fine with the same css as 5.2
I have some doubts 🤔 ?

File: eyed/SECURITY.md (hunk 1 above)
Comment (febryars33) [reply eyed/SECURITY.md:1]: Yes sure

File: eyed/theme.json (hunk 3 above)
Comment (febryars33) [reply eyed/theme.json:8]: that is true haha 😄
change to 5.1 ?

Comment (williamdes) [reply eyed/theme.json:8]: It's a 5.1 only theme ?
On what version did you test it and develop it ?

Comment (febryars33) [reply eyed/theme.json:8]: I use phpmyadmin 5.2

Comment (williamdes) [reply eyed/theme.json:8]: Then it's best to support 5.2
I doubt your CSS can support both versions at a time given the differences between the versions :)

Comment (febryars33) [reply eyed/theme.json:8]: I'll develop 5.2, that same with my phpmyadmin version. and i'll change support to 5.2
If css bugs found, open the issues on my repo
https://github.com/febryars33/eyed-phpmyadmin-theme/issues or this repo :D

what do you think?

Comment (febryars33) [reply eyed/theme.json:8]: and i'll remove 5.1 from theme.json

**Comment by febryars33** [comment]
@williamdes
The theme is now updated and some fixes. Now only version 5.2 😁✌️

**Comment by williamdes** [comment]
> @williamdes The theme is now updated and some fixes. Now only version 5.2 😁✌️

Thanks, I did not have the time to review it but this is still on my todo list :)

**Comment by febryars33** [comment]
Ohh, That's okay 👍

**Review by williamdes — APPROVED** [review]
Hi !
I tested your theme, it's beautiful !

Could you remove some protections so I can force push your branch ?
```
remote: error: GH006: Protected branch update failed for refs/heads/master.
remote: error: Cannot force-push to this protected branch At least 1 approving review is required by reviewers with write access.
```

For future updates please check out the README where it says it's best to use another branch than `master` (on your fork) for contributions to the master branch

In my force push I re-built the css files and removed package.json

**Comment by williamdes** [comment]
Can you remove the branch from the protected branches in the repository settings?

**Comment by febryars33** [comment]
Done

**Comment by febryars33** [comment]
image
image
image


I want to change the icons, but it still points to the folder pmahomme https://localhost/phpmyadmin/themes/pmahomme/img/designer/FieldKey_small.png

**Comment by williamdes** [comment]
I did some research and I think I found out why

This is a very specific use case and the icon should have the same structure as in the default theme to find it

Here is the code that will be called: https://github.com/phpmyadmin/phpmyadmin/blob/44fe103b9c2f00a7d1476bee426f59e9968cc129/libraries/classes/Theme.php#L318-L333

If you follow the code it goes:
- https://github.com/phpmyadmin/phpmyadmin/blob/f5bb90ebb9bbbac8a02ba981fd147dac0b8ec3b4/libraries/classes/Database/Designer.php#L263
- https://github.com/phpmyadmin/phpmyadmin/blob/95fdd35e2fe1dc304efbe8be96a5a6c1a0357562/templates/database/designer/database_tables.twig#L102
- https://github.com/phpmyadmin/phpmyadmin/blob/2c2127b3696aaaa9dbd85ab0776469f8a925b120/libraries/classes/Twig/AssetExtension.php#L31
- https://github.com/phpmyadmin/phpmyadmin/blob/f5bb90ebb9bbbac8a02ba981fd147dac0b8ec3b4/libraries/classes/Database/Designer.php#L263

**Comment by williamdes** [comment]
The best is that you push all your new work, I will rebase and squash it at once and do a force push. Then merge it.
If you read the README it's recommended not to use `master` branch as the branch for your work in a PR :)
But that's okay for now

I will work on trying to add build tools on this repository so you can drop your package.json etc..

**Review by williamdes — APPROVED** [review]
Perfect !

let's merge it like this, you can `git remote add phpmyadmin git@github.com:phpmyadmin/themes.git; git fetch --all -p; git reset --hard phpmyadmin/master`
And then create a new branch for theme updates
